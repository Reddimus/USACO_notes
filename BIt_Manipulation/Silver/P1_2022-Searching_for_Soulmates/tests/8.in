10
2848606687 2411483090
2163993079 120376181
989331885 4216282471
863624108 3999563393
1212328819 2778397791
3750532722 3817368255
4059661957 259784632
4221283158 1100068686
1576922186 1005945963
578750949 3729720418
