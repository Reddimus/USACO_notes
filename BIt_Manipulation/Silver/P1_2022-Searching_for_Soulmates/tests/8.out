91
79
84
76
86
77
77
75
80
82
